nservicebus/messaging/batched-dispatch.md

```diff
@@ -1,17 +1,17 @@
 ---
 title: Batched message dispatch
-summary: Describes how NServiceBus collects outgoing operations when processing message in order to dispatch them more efficiently.
+summary: Describes how NServiceBus can collect outgoing operations in order to dispatch them more efficiently.
 component: Core
-reviewed: 2022-08-23
+reviewed: 2022-09-20
 ---
 
-In Versions 6 and above, all outgoing operations that happen as part of processing a message (e.g. commands, responses, events) will be bundled together and passed to the transport after message handling pipeline has completed. This has two benefits:
+In Versions 6 and above, all outgoing operations that happen as part of processing a message (e.g. commands, responses, events) will be bundled together and passed to the transport after the message handling pipeline has been completed. With the introduction of the [transactional session](/nservicebus/transactional-session), outgoing message operations can also be bundled together and passed to the transport when the session commits. This has two benefits:
```

Comment:
```suggestion
Starting from Versions 6, all outgoing operations executed during message processing (e.g. commands, responses, events) are bundled together and handed to the transport only after the message handling pipeline has completed ([transactional session](/nservicebus/transactional-session) offers similar behavior for outgoing message operations). This has two benefits:
```

Reply:
I'm unsure about this. I think mentioning the commit is important